backend/server.py

```
@@ -7119,47 +7119,50 @@ async def workflow_orchestrator_status(
-            "failed_retried": st.failed_retried,
-            "alerts_emitted": st.alerts_emitted,
-        }
-    except Exception:
-        pass
+        queue_status = {"max_concurrent": 2, "active": 0, "queued": 0}
+        try:
+            from services.orchestrator_queue import get_orchestrator_queue
+            q = get_orchestrator_queue()
+            queue_status = q.status()
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)